Remove duplicated 'compression' documentation from googlecloudexporter readme

nzbart opened the pull request:
**Description:**
The `compression` metric option appears more than once in the readme for the googlecloudexporter exporter, so I've deleted one of the copies. The first copy has more detail, so I preserved that one.

**Link to tracking Issue:** 
None.

**Testing:**
Manually reviewed readme on GitHub.

**Documentation:**
This PR is a documentation tidy-up.

dashpole reviewed (approved):
Thanks!

codeboten commented:
Thanks @nzbart! Please sign the CLA and we can get this merged

nzbart commented:
@codeboten I am not sure why the PR shows that I haven't signed the CLA. I've received a confirmation email stating that I have signed it, and can download the attached signed PDF.

codeboten commented:
@nzbart not sure but it looks like the PR was opened with a different username than the original commit (nzbart & bartecargo)? maybe that's causing an issue

nzbart committed:
Remove duplicated 'compression' documentation

The compression metric option appears more than once in the readme, so
I've deleted one of the copies. The first copy has more detail, so I
preserved that one.

nzbart commented:
@codeboten well spotted! I've fixed the problem and the CLA is signed.
